src/Backend/Core/Engine/TemplateModifiers.php

```diff
@@ -307,4 +307,16 @@ public static function showBool($status, $reverse = false)
 
         return $status;
     }
+
+    /**
+     * Returns the count of the count of the array.
+     *
+     * @param array $data
+     *
+     * @return int
+     */
+    public static function count(array $data)
+    {
+        return count($data);
```

Comment:
I would wait for the twig integration ;) modifiers are completely reworked in there ;)